src/pages/SubmissionConfig.js

```diff
@@ -47,7 +50,7 @@ export const SubmissionConfig = ({ methodId }) => {
       const searchResult = await Ajax(signal).Wds.search.post(recordType)
       setRecords(searchResult.records)
     } catch (error) {
-      notify('error', 'Error loading WDS records', { detail: await (error instanceof Response ? error.text() : error) })
+      setNoRecordTypeData(`Data Table not found: ${recordType}`)
```

Comment:
I think we're referring to a table, not the product

```suggestion
      setNoRecordTypeData(`Data table not found: ${recordType}`)
```